### PR TITLE
[docs] add DisjunctiveProgramming

### PR DESCRIPTION
**Pull request by odow**
Probably going to need a few fixes 

cc @pulsipher 
cc @hdavid16 

https://jump.dev/JuMP.jl/previews/PR3598/packages/DisjunctiveProgramming/

### CODE REVIEW

**Comment by pulsipher** [comment]
Thanks @odow, I suspected this wouldn't quite work. I think the best way forward would be for me to make a JuMP specific readme like I did for InfiniteOpt. I should have time to do that this week.

**Comment by joaquimg** [comment]
Another option would be to cleanup the readme since most of it is replicated in the docs.

**Comment by pulsipher** [comment]
> Another option would be to cleanup the readme since most of it is replicated in the docs.

This along with improving the docs is on my TODO list, I just need to find the time :)

**Comment by pulsipher** [comment]
I think I fixed the vale errors

File: docs/packages.toml [hunk 1]
```diff
@@ -125,6 +125,11 @@
 [DAQP]
     user = "darnstrom"
     rev = "v0.5.2"
+[DisjunctiveProgramming]
+    user = "hdavid16"
+    rev = "b71f715ba826fa63677da9590ae41b41ca5024e6"
```
Comment (odow) [inline docs/packages.toml:130]:
```suggestion
    rev = "5d814e9023acf25672977632bc9dc111d68ea8ca"
```

File: docs/packages.toml [hunk 2]
```diff
@@ -125,6 +125,11 @@
 [DAQP]
     user = "darnstrom"
     rev = "v0.5.2"
+[DisjunctiveProgramming]
+    user = "hdavid16"
+    rev = "5d814e9023acf25672977632bc9dc111d68ea8ca"
```
Comment (odow) [inline docs/packages.toml:130]:
```suggestion
    rev = "6a4d0ac4a7484e52b1dbff9cee310a73d2d23e81"
```

**Comment by odow** [comment]
Merging because we can always update later. I don't see the typos @pulsipher is talking about.

**Pull request merged**